### ISSUE TITLE
whatlang-0.3.2 beta regression

### ISSUE THREAD

**Issue by est31** [rust-lang/rust#43685]
[whatlang](https://crates.io/crates/whatlang) version 0.3.2 regressed from stable to beta ([full log](https://cargobomb-reports.s3.amazonaws.com/stable-1.19-beta-1.20.0-beta.1/beta-2017-07-18/reg/whatlang-0.3.2/log.txt)):

```
Aug 04 03:49:14.070 INFO kablam! error: failed to parse manifest at `/source/Cargo.toml`
Aug 04 03:49:14.070 INFO kablam! 
Aug 04 03:49:14.070 INFO kablam! Caused by:
Aug 04 03:49:14.070 INFO kablam!   can't find `example` bench, specify bench.path
```
cc @greyblake

**Comment by greyblake** [comment]
Hey can I help you?
I am not sure I understand the issue.
The only diff in `Cargo.toml` between 0.3.1 and 0.3.2 is that in `0.3.2` was added external dependency:

```
fnv = "1.0"
```

Could it be really the origin of the problem?

**Comment by est31** [comment]
@greyblake its an issue with cargo where a newer version of cargo doesn't accept your Cargo.toml. We are in the process of finding out what the cause is. Your crate is not the only one affected, see this link: https://github.com/rust-lang/cargo/issues/4365

**Comment by alexcrichton** [comment]
Fixed by https://github.com/rust-lang/cargo/pull/4389

**Comment by est31** [comment]
Still not fixed in 1.20.0-beta.2: http://cargobomb-reports.s3.amazonaws.com/stable-1.19-beta-1.20.0-beta.2/beta-2017-08-13/reg/whatlang-0.5.0/log.txt

**Comment by alexcrichton** [comment]
Not a regression I believe, the failure mode just changed. Depending on the crate works and otherwise the packaged version has an invalid manifest.
